pkg/template/master_template.go

```diff
@@ -427,7 +427,8 @@ storage:
       filesystem: root
       mode: 0644
       contents:
-        source: "data:text/plain;charset=utf-8;base64,{{  index .Files "k8s-resource/calico-crds.yaml" }}"
+        source: "s3://gs-tc-crds/calico/v3.16.5/crds.yaml"
```

Comment:
Is this public object? How are we going to upload those?

Reply:
It's a public object. I created the bucket. I don't expect this to change often, but if we need to change it, we would upload a new one with a different version and update this line here to match. It's clunky, but shouldn't need to happen more than a few times a year.

Reply:
Are those unchanged upstream CRDs? Do we have instructions how to download them?